### ISSUE TITLE
[Feature Request] Change config directory location

### ISSUE THREAD

**Issue by nicman23** [arfbtwn/banshee#60]
I have a samba share auto connecting on my music folder and basically have 2 versions for my music

-> high quality opus for my laptop as it has a small ssd that are encoded from
-> flacs

I would like to be able to have the config / db folder on the same folder for when I am in my home vs when I am outside so I don't have to rescan my library every time.

**Comment by arfbtwn** [comment]
Yeah I've been considering a similar feature for a different purpose actually, I have lots of language files that I'd like to have organised but I don't want them in my music library and I don't want Banshee starting Unit 1 of Italian for Beginners when its in Party Mode 😆 

So, the idea I've been toying with is to have an "Add Library" feature in the source list instead of just having Banshee defined things like Music, Video, Podcasts and Devices.
